tests/cp/test_cp_halo.py

```
@@ -0,0 +1,142 @@
+    Dq, Dk, Dv = 3, 4, 5
+
+    # Create deterministic inputs per rank
+    torch.manual_seed(2025 + rank)
+    q_in = torch.randn(B, T_local, Dq)
+    k_in = torch.randn(B, T_local, Dk)
+    v_in = torch.randn(B, T_local, Dv)
+
+    # No varlen here
+    cu_seqlens = None
+
+    q_ext, k_ext, v_ext, cu_ext = halo_exchange_and_extend(
```

Ruff (0.14.5)

Unpacked variable `k_ext` is never used

Prefix it with an underscore or any other dummy variable pattern

(RUF059)